src/Appwrite/Utopia/Response/Model/Execution.php

```diff
@@ -65,6 +65,12 @@ public function __construct()
                 'default' => '',
                 'example' => '',
             ])
+            ->addRule('stdout', [
```

Comment:
We need to hide this attribute for client side sdks 🤔

Reply:
It is set to an empty string on controller-level before returning a response. Should be in all places, I double-checked.